01-make_graph.py

```diff
@@ -49,13 +49,15 @@ def parsed_meta_yaml(text):
 with open('bad.txt', 'r') as f:
     bad = set(f.read().split())
 
+new_bad = []
+
 print(bad)
 print('reading graph')
 gx = nx.read_gpickle('graph.pkl')
 # gx = nx.read_yaml('graph.yml')
 
 new_names = [name for name in names if
-             name not in gx.nodes and name not in bad]
+             name not in gx.nodes]
```

Comment:
Do we still need to read bad?

Reply:
No. Previously, even if a recipe changed it will still stay in `bad`. Now it's not.

Reply:
Would you mind removing the bad reading from above?